src/components/ElementBar/TextAlignmentButtons/TextAlignmentButtons.tsx

```diff
@@ -35,15 +35,8 @@ export function TextAlignmentButtons() {
   const slideInstance = useWhiteboardSlideInstance();
 
   const handleRadioClick = useCallback(
-    (event: ChangeEvent<HTMLInputElement>) => {
-      if (!event.target.checked) {
-        return;
-      }
-
-      const updates = calculateTextAlignmentUpdates(
-        elements,
-        event.target.value as TextAlignment,
-      );
+    (textAlignment: TextAlignment) => {
+      const updates = calculateTextAlignmentUpdates(elements, textAlignment);
```

Comment:
It should only update if at least one element has a different alignment.
Otherwise, if you click it, then undo looks like nothing happens.

![alignmnt](https://github.com/nordeck/matrix-neoboard/assets/6216686/9708fae3-5900-4d69-b5a2-6e0428a97f40)